Feature/logout and bottom sheet

baraa0abd opened the pull request:
## Editing Bottom sheet for signin ##

<img width="364" height="246" alt="image" src="https://github.com/user-attachments/assets/c6c66c56-de8b-4d07-a53b-d5b4647e5220" />

baraa0abd committed:
bottom sheet component

baraa0abd committed:
Refactor bottom sheet layout and add string resources

- Update `bottom_sheet.xml`:
    - Change `textColor` for title, description, cancel button, and "Go to Website" button to use color resources.
    - Change `text` for cancel button and "Go to Website" button to use string resources.
    - Remove commented out titles for some sections.
- Add `go_to_website` string resource to `values/strings.xml` and `values-night/strings.xml`.

baraa0abd committed:
Update bottom sheet UI and add button backgrounds

- Add `ic_account_circle_64dp.xml`:
    - Layer-list with an oval shape and a centered icon.
    - Oval shape has `brand_tertiary` background color and 64dp size.
- Update `bottom_sheet.xml`:
    - Set background of the top `View` to `ic_account_circle_64dp`.
    - Set background of the "Cancel" `TextView` to `button_secondary_background`.
    - Set background of the "Go to website" `TextView` to `button_primary_background`.
- Add `button_primary_background.xml`:
    - Shape with 12dp corner radius and `brand_primary` solid color.
- Add `button_secondary_background.xml`:
    - Shape with 12dp corner radius.

baraa0abd committed:
Add drag handle and update line color in bottom sheet

- Add a `View` for the drag handle in `bottom_sheet.xml`.
- Set background of the drag handle to `bottom_line`.
- Change solid color in `bottom_line.xml` from `#4A4B5C` to `@color/shade_quaternary`.

baraa0abd committed:
Update font style for buttons in bottom sheet

- Apply `style="@style/Typography.title_xl"` to the cancel and "go to website" buttons.
- Remove `android:fontFamily="sans-serif-medium"` from both buttons.

MuhammedWael9991 commented on app/src/main/res/drawable/bottom_line.xml:
```diff
@@ -2,13 +2,13 @@
     <item>
         <shape>
-            <solid android:color="@color/background_color" />
+            <solid android:color="#1B1C2A" />
```
Please use the design system color called 'background_card

MuhammedWael9991 commented on app/src/main/res/drawable/bottom_rounded_corners.xml:
```diff
@@ -1,5 +1,9 @@
+        android:bottomRightRadius="12dp"
+        android:topLeftRadius="0dp"
+        android:topRightRadius="0dp" />
+    <solid android:color="#1B1C2A" />
```
same here please

MuhammedWael9991 commented on app/src/main/res/drawable/bottom_sheet_corners.xml:
```diff
@@ -1,8 +1,8 @@
+        android:topLeftRadius="20dp"
+        android:bottomLeftRadius="0dp"
+        android:bottomRightRadius="0dp" />
+    <solid android:color="#1B1C2A" />
```
and here

MuhammedWael9991 commented on app/src/main/res/layout/bottom_sheet.xml:
```diff
@@ -0,0 +1,76 @@
+        android:layout_height="wrap_content"
+        android:layout_gravity="center_horizontal"
+        android:layout_marginBottom="8dp"
+        android:fontFamily="sans-serif-medium"
```
Please use the fontStyle from the design system, like 
'android:style="@style/Typography.title_sm"'
instead of fontFamily, textSize, and textStyle

MuhammedWael9991 commented on app/src/main/res/layout/bottom_sheet.xml:
```diff
@@ -0,0 +1,76 @@
+        android:text="Let's get you set up! We'll take you to the website to create your account."
+        android:textColor="@color/shade_secondary"
+        android:textSize="12sp"
+        android:fontFamily="sans-serif" />
```
same here

baraa0abd committed:
Merge remote-tracking branch 'origin/develop' into feature/logout-and-bottomSheet

# Conflicts:
#	app/src/main/res/values/strings.xml

baraa0abd committed:
Update bottom sheet text and colors

- Update text for `join_cineverse` and description in `bottom_sheet.xml` to use string resources.
- Update font family for title to `@style/Typography.title_sm` and description to `@style/Typography.body_lg_medium`.
- Change solid color in `bottom_line.xml` from `#1B1C2A` to `@color/shade_quaternary`.
- Change solid color in `bottom_sheet_corners.xml` and `bottom_rounded_corners.xml` to `@color/background_bottom_sheet_card`.
- Add `join_cineverse` and `let_s_get_you_set_up_we_ll_take_you_to_the_website_to_create_your_account` string resources to `values/strings.xml` and `values-night/strings.xml`.

MuhammedWael9991 commented on app/src/main/res/drawable/ic_account_circle_64dp.xml:
I suggest using an icon from the design system

baraa0abd committed:
Merge remote-tracking branch 'origin/develop' into feature/logout-and-bottomSheet

# Conflicts:
#	app/src/main/res/values-night/strings.xml
#	app/src/main/res/values/strings.xml
